src/daemon/rpmostreed-os-experimental.cxx

```diff
@@ -165,11 +165,65 @@ osexperimental_handle_live_fs (RPMOSTreeOSExperimental *interface,
   return TRUE;
 }
 
+static gboolean
+osexperimental_handle_search (RPMOSTreeOSExperimental *interface,
+                              GDBusMethodInvocation *invocation,
+                              GPtrArray *queries)
```

Comment:
The type of this argument would be something like `const char * const *`. You can see the prototype in `rpm-ostreed-genenerated.h`.